packages/shared/src/types/index.ts

```diff
@@ -336,6 +336,7 @@ export type RemoteSearchOptions = SearchHooks & {
   domain?: string;
   indexName: string;
   searchIndexes?: RemoteSearchIndexInfo[];
+  searchLoading?: boolean;
```

Comment:
should we document this new option?

Reply:
remote search API is not stable and in long term will write a doc for it.